src/components/Button/Button.styles.ts

```diff
@@ -301,6 +302,30 @@ const StyledButton = styled.button<{
     }
   `}
 
+  ${({theme, subtype, variant, isLoading}) =>
+    variant === "secondary" &&
+    subtype === "inverse" &&
+    `
+    color: white;
+    background-color: transparent;
+    border: ${theme.borderWidth.md} solid white;
+
+    &:hover:not(:disabled) {
+      background-color: rgba(255, 255, 255, 0.05);
```

Comment:
Question; Is there a theme prop we can use here?

Reply:
For this one, according to Lou there is no token we can use at the moment